src/isPlainObject.js

```diff
@@ -0,0 +1,20 @@
+const fnToString = (fn) => Function.prototype.toString.call(fn)
```

Comment:
There is already a module for this. https://github.com/garbles/yolk/blob/master/src/isFunction.js